drivers/counter/counter_ace_v1x_art_regs.h

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright (c) 2022 Intel Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef __COUNTER_ACE_V1X_ART_REGS__
+#define __COUNTER_ACE_V1X_ART_REGS__
+
+#if CONFIG_ACE_V1X_ART_COUNTER
+
+struct DfTSCTRL {
+	union {
+		uint32_t full;
+		struct {
+			uint32_t    cdmas :  5;
+			uint32_t     odts :  1;
+			uint32_t     lwcs :  1;
+			uint32_t    hhtse :  1;
+			uint32_t    rsvd9 :  2;
+			uint32_t    clnks :  2;
+			uint32_t    dmats :  2;
+			uint32_t   rsvd29 : 16;
+			uint32_t    ionte :  1;
+			uint32_t      ntk :  1;
+		} part;
+	} u;
+};
+
+union DfDWCCS {
+	uint64_t full;
+	struct {
+		uint64_t      dwc : 64;
+	} part;
```

Comment:
You're unioning a uint64_t with... a struct containing exactly one uint64_t.  Why?